README.md

```diff
@@ -1,2 +1,39 @@
-# provider-operator-example
-The GitHub repository provides a operator example for integrating database providers with the OpenShift Database Access/DBaaS Operator. The examples are intended to help developers understand how to create their operator and use the operator with DBaaS operator.
+# Provider Operator Example
+The GitHub repository provides a operator example for integrating database providers with the OpenShift Database Access/DBaaS Operator. The examples are intended to help developers understand how to create their operator and use the operator to with DBaaS operator.
+
+# Create your Operator 
+
+``` 
+operator-sdk init --domain redhat.com --repo github.com/RHEcosystemAppEng/provider-operator-example.git
+
+operator-sdk edit --multigroup=true
+
+
+operator-sdk create api --group dbaas --version v1beta1 --kind ProviderInventory --resource --controller
+operator-sdk create api --group dbaas --version v1beta1 --kind ProviderConnection --resource --controller
+operator-sdk create api --group dbaas --version v1beta1 --kind ProviderInstance --resource --controller
+
+```
+
+## Import DBaaS operator api in your Operator
+
+In your go.mod and added spec status in `providerconnection_types.go` 
```

Comment:
Add the following to your go.mod file:
github.com/RHEcosystemAppEng/dbaas-operator v0.4.1-0.20230403142057-6112a98be1a6
and add DBaaSInventorySpec/DBaaSInventoryStatus, DBaaSInstanceSpec/DBaaSInstanceStatus, and DBaaSConnectionSpec/DBaaSConnectionStatus as the Spec/Status to the ProviderInventory, ProviderInstance, and ProviderConnection data structures, respectively.